### DISCUSSION TITLE
点击抢票之后会出现这个未知错误的情况 不知道是哪出了问题 希望大佬救救QAQ

### DISCUSSION THREAD

**Discussion by Gofish216** [mikumifa/cppTickerBuy#17] [提问专区]
![94A7F83DB2C6B636F3143A2A7A9D935A](https://github.com/user-attachments/assets/34e5adb8-1c95-4838-b11b-dc78c1b21b3e)

**Comment by csr050801** [comment]
我也是同样问题人有点麻了

**Reply by SkyFall5Yzy** [reply]
为什么我连选票时间都没有啊？那一栏都是空的

**Comment by mikumifa** [comment]
风控了，换网络（例如换梯子节点，连手机热点，换电脑，插拔路由器
让抢票时候记得发包慢一点

**Reply by Stein-er** [reply]
如果是214也是风控吗

2025-05-14 19:59:57.846 | INFO     | tab.go:inner_request:214 - 状态码: False(抢票失败), 请求体: {'isSuccess': False, 'errorCode': -99999, 'message': '系统繁忙，请稍后再试'}

**Reply by mikumifa** [reply]
cpp开慢点一直发请求就行，cpp这些报错都意义不明
